src/Symfony/Component/Notifier/Bridge/Lox24/Lox24Transport.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
```

Comment:
Please remove this line, and check other files in the PR.

Reply:
@welcoMattic remove `declare(strict_types=1);` from every class?

Reply:
done

Reply:
This is **not** done. There are still such lines in the diff of the PR.

Reply:
Done now